gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -185,14 +188,21 @@ export default function NxGlobalSidebarPage() {
                   expanded view, but will not be displayed in the collapsed view.
                 </NxTable.Cell>
               </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell><NxCode>nx-global-sidebar__other-content</NxCode></NxTable.Cell>
+                <NxTable.Cell>element</NxTable.Cell>
+                <NxTable.Cell>
+                  A simple container for content that will appears below the navigation and above the footer.
+                </NxTable.Cell>
+              </NxTable.Row>
             </NxTable.Body>
           </NxTable>
```

Comment:
I think we need to document on this page that there is a footer component and that it is required.

Reply:
> I think we need to document on this page that there is a footer component and that it is required.

https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/cb864450f5887f39b15466869fae4bc868842500